src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt

```diff
@@ -43,6 +43,7 @@ class PrivacyModule(
                 .filter { it.created.isAfter(lastRun) }
                 .filter { it.visibilityType == null }
                 .filter { it.body?.matches(patterns) ?: false }
+                .filterNot { it.getAuthorGroups()?.any { it == "staff" } ?: false }
```

Comment:
```suggestion
                .filterNot { it.getAuthorGroups()?.any { it == "helper" || it == "staff" } ?: false }
```

Reply:
I thought only Mojang employees were going to be exempt from this?

Reply:
All helpers and mods should be too